src/semidiscretization/semidiscretization_euler_plasma.jl

```diff
@@ -0,0 +1,539 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+struct ParametersEulerPlasma{RealT <: Real, TimestepPlasma}
```

Comment:
Just to have this posted somewhere: We should add at some point docstrings to these structs where we explicitly mention the paper we take the test cases from to familiarize people with the equations we use.